tests/integration-tests/tests/arm_pl/test_arm_pl.py

```diff
@@ -58,13 +61,22 @@ def _test_armpl_examples(
     for module in [armpl_module_general_name, armpl_module_name, gcc_module_name]:
         assert_that(module_result).contains(module)
 
+    # Check that EULA docs are correctly linked in the module output
+    eula_path = re.search(".*EULA can be found in the '(.*)'", module_result)[1]
+    # Clean up ANSI escape sequences from the output
+    eula_path = re.compile(r"\x1b[^m]*m| \x08").sub("", eula_path)
+    eula_path_result = remote_command_executor.run_remote_command("ls {0}".format(eula_path)).stdout
+    assert_that(eula_path_result).contains("license_agreement.txt")
+    assert_that(eula_path_result).contains("third_party_licenses.txt")
+
     # Assert pass the example tests
     remote_command_executor.run_remote_command(
-        f"sudo chmod 777 /opt/arm/armpl/{armpl_version}/armpl_{armpl_version}_gcc-{gcc_version}/examples"
+        f"sudo chmod 777 /opt/arm/armpl/{armpl_version}/armpl_{armpl_major_minor_version}_gcc-{gcc_version}/examples"
```

Comment:
Is there a way to determine we are using major_minor version or full version? So we needn't change test again in next time. For example, check the patch version of full version. If it is 0, then we use major_minor_version, otherwise we use full version

Reply:
but it is fine since we need to change module file as well.